client/src/components/Bookings/BookingsSideBanner/useStyles.ts

```diff
@@ -0,0 +1,7 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {},
```

Comment:
Please delete this file if we don't intend to use it?

Reply:
done